File: pyocel/examples/import_jsonocel_export_xmlocel.py
import ocel


def execute_script():
    print("validated input", ocel.validate("../logs/minimal.jsonocel", "../schemas/schema.json"))
    log = ocel.import_log("../logs/minimal.jsonocel")
    ocel.export_log(log, "log1.xmlocel")
    print("validated output", ocel.validate("log1.xmlocel", "../schemas/schema.xml"))
    ocel.import_log("log1.xmlocel")


if __name__ == "__main__":
    execute_script()

File: ocel/src/main/java/it/unibz/inf/pm/ocel/examples/ImportJsonocelExportXmlocel.java
package it.unibz.inf.pm.ocel.examples;

import it.unibz.inf.pm.ocel.OcelInitial;
import org.dom4j.DocumentException;

public class ImportJsonocelExportXmlocel {

    public static void main(String[] args) throws Exception {
        try {
            System.out.print("validated input:");
            OcelInitial.validate("ocel/logs/minimal.jsonocel", "ocel/schemas/schema.json");
            Object log = OcelInitial.import_log("ocel/logs/minimal.jsonocel","toXML");
            OcelInitial.export_log(log,"ocel/examples/log1.xmlocel");

            System.out.print("\nvalidated output:");
            OcelInitial.validate("ocel/examples/log1.xmlocel", "ocel/schemas/schema.xml");
        } catch (DocumentException e) {
            e.printStackTrace();
        }
    }
}
